src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/request/UpdateReportRequest.kt

```diff
@@ -24,6 +24,9 @@ data class UpdateReportRequest(
   val reportedBy: String? = null,
   @Schema(description = "When the incident report was created", required = false, defaultValue = "null", example = "2024-04-29T12:34:56.789012")
   val reportedAt: LocalDateTime? = null,
+
+  @Schema(description = "Whether the parent event should also be updated", required = false, defaultValue = "false", example = "true")
+  val updateEvent: Boolean = false,
```

Comment:
report deletion already has an option to propagate to the parent event, so it makes sense to allow propagating field updates. there are no other fields in the event than those already listed here – we might no need any special event-editing endpoints!

of course, the parent event is very likely to change shape and be renamed in future